lib/src/main/java/com/diffplug/spotless/JarState.java

```diff
@@ -107,10 +147,4 @@ public ClassLoader getClassLoader() {
 	public ClassLoader getClassLoader(Serializable key) {
 		return SpotlessCache.instance().classloader(key, this);
 	}
-
-	/** Returns unmodifiable view on sorted Maven coordinates */
-	@Deprecated
-	public Set<String> getMavenCoordinates() {
-		return Collections.unmodifiableSet(mavenCoordinates);
-	}
```

Comment:
I suppose it was already the plan to bump the major version of `lib` and mention the removal of this method in the changelog, wasn't it? :)

Reply:
I appreciate the callout!